frontend/src/app/dynamic-form/dynamic-form.component.ts

```diff
@@ -51,6 +54,28 @@ export class DynamicFormComponent implements OnInit {
             timestamp: new Date()
         };
 
-        this.eventEntityService.saveEventUsingPOST(event).subscribe();
+        this.eventEntityService.saveEventUsingPOST(event).subscribe(() => this.openDialog());
+
+    }
+
+    openDialog(): void {
+        const dialogRef = this.dialog.open(FormSubmissionDialogComponent, {
+            width: '250px',
```

Comment:
You might be able to make the popup handle the header text on the same line by increasing the width.

Perhaps doing something like
```
max-width: 400px;
width: auto;
```

That should allow the popup to be as wide as 400px, but if it's on a mobile screen it will fit to that screen instead.